### DISCUSSION TITLE
[BUG] GUI is not updating "height" for connected peers and lags behind

### DISCUSSION THREAD

**Discussion by boettges** [Chia-Network/chia-blockchain#5134] [Support]
**Describe the bug**
Despite new block hashes coming in timely and frequently and the overall status of my node is disaplyed as "synced" in the GUI, the displayed "height" for connections to peers often lags behind.
It seems as if the height for peer connections is not updating at least in the Windows version of the app (1.1.5).

Example:
The "peak height" of my "full node status" could be at 285575, but the height of connected nodes remains stuck at 284511.

**To Reproduce**

1. Use the GUI of 1.1.5 for windows
2. Wait for the node to sync
3. Observe the height of your own "full node status" drifting apart from the heights displayed for peer connections to other nodes in the "connections" section of the "full node" tab.
4. If you use `chia show -c` the heights are properly displayed.
5. If you hit the "keys" tab and switch back to the full node (still active in the background) by clicking on your private key the connections' heights will be updated due to a **full GUI refresh**.

**Expected behavior**
Connection height is updated live.

**Desktop (please complete the following information):**

- OS: Windows 10 Pro

I know that this is not the most critical bug to fix right now. Yet, the displayed difference between the heights can give the impression that the node might be connected to "lagging" peers. This leads to a lot of confusion amongst those that suspect faulty connections being the reason for not finding the proof of a block. 

Thanks a lot in advance!

**Comment by gromran** [comment]
> displayed "height" for connections to peers often lags behind

not a bug, only a delayed update (or not synced peer wallets)

**Reply by boettges** [reply]
The peer nodes remain in sync as can be confirmed via CLI as explained above.

Since every new incoming block is causing an incremental interface update anyways, I do not see any reason why the displayed height can't be updated as well. This would mitigate a lot of confusion about whether a node has valid, updated and working peer connections.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
